apps/native-component-list/src/screens/Location/BackgroundLocationMapScreen.tsx

```diff
@@ -84,15 +82,6 @@ export default function BackgroundLocationMapScreen(props: Props) {
     })();
   }, [props.navigation]);
 
-  if (!permission) {
-    return (
-      <Text style={styles.errorText}>
-        Location permissions are required in order to use this feature. You can manually enable them
-        at any time in the "Location Services" section of the Settings app.
-      </Text>
-    );
-  }
-
```

Comment:
This flow seems useful to handle the case where we've asked for permissions and were denied.

Reply:
added back (although we're now checking for foreground location permission here because the feature still works with just that permission, it just won't receive any updates when the app is backgrounded. And we log warnings if that's the case)